documentation-site/pages/components/datepicker.mdx

```diff
@@ -67,6 +67,7 @@ A simple and reusable component to work with or select a date or a range of date
 </Example>
 
 The `formatString` property supports all the patterns supported by [date-fns' format method](https://date-fns.org/v2.8.1/docs/format).
+For example, if someone wants to change it to Australian format, one has to simply pass the `formatString={'DD/MM/YYYY'}` as the props.
```

Comment:
should this be `d/MM/YYYY`? DD stands for day of year, if I am correct: https://date-fns.org/v2.8.1/docs/format

Reply:
@gergelyke 
Yes. Changed it for the better reference. Please have a look